cli/src/cmd/forge/build.rs

```diff
@@ -24,10 +24,17 @@ use watchexec::config::{InitConfig, RuntimeConfig};
 // Loads project's figment and merges the build cli arguments into it
 impl<'a> From<&'a BuildArgs> for Figment {
     fn from(args: &'a BuildArgs) -> Self {
-        let figment = if let Some(root) = args.root.clone() {
-            Config::figment_with_root(root)
+        let figment = if let Some(ref config_path) = args.config_path {
+            if !config_path.exists() {
+                panic!("error: config-path `{}` does not exist", config_path.display())
+            }
+            if !config_path.ends_with(Config::FILE_NAME) {
```

Comment:
Is this necessary? Why not allow any kind of filename?

Reply:
mirrored `cargo --manifest-path` behaviour here. this will also ensure that this specific file is picked up by `Config::load_with_root` which expects a foundry.toml

Reply:
Got it - and I guess there's no expected use case where somebody would have 2 foundry toml files?

Reply:
this usecase should be covered via profiles